docs/custom-workload.md

```diff
@@ -0,0 +1,92 @@
+# Deploying a custom workload
+
+## Example workload
+
+A typical windows deployment would look like this.
+
+```shell script
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: win-webserver
+  name: win-webserver
+spec:
+  selector:
+    matchLabels:
+      app: win-webserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: win-webserver
+      name: win-webserver
+    spec:
```

Comment:
please add [PodOS](https://github.com/openshift/windows-machine-config-operator#running-windows-workloads) to this deployment
```yaml
...
    spec:
      os:
        name: "windows"
      tolerations:
      ...
```